src/providers-registry/providers-registry.contract.ts

```diff
@@ -205,6 +206,14 @@ export async function handle(state: ProvidersRegistryState, action: ProvidersReg
       return {state};
     }
 
+    case "providerAdmins": {
```

Comment:
this function should be added to "getFunctions" in `src/providers-registry/types.ts`